python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -629,6 +645,85 @@ def unpack(
         check.failed(f"Unexpected class {self.klass}")
 
 
+@whitelist_for_serdes
+class SkipOnNotAllParentsUpdatedRule(
+    AutoMaterializeRule,
+    NamedTuple(
+        "_SkipOnNotAllParentsUpdatedRule", [("require_update_on_all_parent_partitions", bool)]
+    ),
+):
+    """An auto-materialize rule that enforces that an asset can only be requested for execution
```

Comment:
Is "request for execution" the language that we use in other auto-materialize policy docstrings? I think we should try our best to be consistent.

Reply:
Looks like we just say "asset can only be materialized"--I've updated this